database-lin.lisp

```diff
@@ -29,6 +29,30 @@
    ( 6 :dpad-h)
    ( 7 :dpad-v)))
 
+;; same as "xbox-360", all buttons where right in the visualizer
```

Comment:
This is exactly the same as the already existing xbox-360 definition, as even the IDs are the same.